tools/release/image-mirror/config.yaml

```diff
@@ -1,8 +1,8 @@
 ## The source repositories and target repositories are one-to-one correspondence.
 sourceRepos:
-  - registry: kubebuilder
+  - registry: brancz
     name: kube-rbac-proxy
-    host: gcr.io
```

Comment:
I am worried that this may overwrite previously mirrored image tags. Should we instead create a new `sourceRepo` entry where the host is different and remove the `v0.14.2` tag from this registry entry?

Reply:
Oh, this is a good point. In this case, I would prefer that we only include `v0.14.2` and remove the [rest of tags](https://github.com/aws-observability/aws-otel-collector/blob/main/tools/release/image-mirror/config.yaml#L8-L12) so it will only mirror `0.14.2` leaving the rest of them not impacted.